### ISSUE TITLE
Unable to set up work profile on LOS 17

### ISSUE THREAD

**Issue by dimqua** [PeterCxy/Shelter#90]
The app just stuck in the process of creation new work profile for Shelter. If I kill and restart, Shelter suggests to remove old profile and create a new one. And this doesn't work again.

**Comment by nunbit** [comment]
> The app just stuck in the process of creation new work profile for Shelter. If I kill and restart, Shelter suggests to remove old profile and create a new one. And this doesn't work again.
same

**Comment by nunbit** [comment]
> The app just stuck in the process of creation new work profile for Shelter. If I kill and restart, Shelter suggests to remove old profile and create a new one. And this doesn't work again.

same

**Comment by PeterCxy** [comment]
@nunbit Did you see a notification asking you to click in order to set up Shelter? That notification is now needed because Android 10 blocked Shelter from bringing itself up inside the work profile (which is actually a good change limiting apps' background behavior). If you don't click the notification, then basically it won't work. I guess you may have enabled no-disturb mode so you missed that notification?

**Comment by nunbit** [comment]
Yes you are right. Someone told me that just yesterday. It was disturb mode. Thank you. Solved.
